docs/guides/repositories_in_project.md

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "Guides"
+---
+# Adding repositories to the project
+
+The guide provides an information and the example on how to add repositories to the project. 
+
+## Artifactory behavior
+
+The attribute `project_environments` (`environments` in the API call) is ignored by Artifactory, if the repository is not assigned to the existing project.
+That attribute can only be set to the repository if it's assigned to the project already. 
+The project can't be created with the list of non-existing repositories, and the repository can't be assigned to non-existing project.
+Thus, if the project and the repository/repositories are created at the same time in one Terraform configuration, we have a state drift 
+for the `project_environments` attribute.
+
+This is happening, because the repositories need to be created first, then the project with the list of repositories gets 
+created. Since `project_environments` attribute is ignored in the first step, we will only have this attribute in the Terraform state, and
+not in the actual repository properties.
+
+On the next step, when the repo gets assigned to the project by the project resource, the default value `DEV` is assigned to 
+repositories' `project_environments` attribute. If the desired value on the first step was `DEV`, then the values match and no state
+drift occurs. But if the desired value was `PROD`, we will get an error message when updating the config by `terraform plan`/`terraform apply`.
+
+```
+  # artifactory_local_docker_v2_repository.docker-v2-local will be updated in-place
+  ~ resource "artifactory_local_docker_v2_repository" "docker-v2-local" {
+        id                       = "myproj-docker-v2-local"
+      ~ project_environments     = [
+          - "DEV",
+          + "PROD",
+        ]
+    } 
+```
+
+## Workaround
+
+~> **Note:** In the Project provider documentation, we strongly recommend using the `repos` attribute to manage the list of repositories.
```

Comment:
I don't think you need the `**Note:**` since `~>` will be in a yellow box. And the accessibility text will automatically include `**Note:**`. This may have 2 of these now.

See https://developer.hashicorp.com/terraform/registry/providers/docs#callouts

> For colorblind users (and for clarity in general), callouts will automatically start with a strong-emphasized word to indicate their function.

Reply:
yeah, removing `**Note:**` doesn't change the appearance: 
<img width="685" alt="image" src="https://user-images.githubusercontent.com/55418063/207434000-c26d5d74-ba5e-492b-a354-a690ba9d84b3.png">
Removed both.